README.md

```diff
@@ -885,6 +891,51 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## Installing Python 3.8.3 on macOS BigSur
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+Users of macOS 11 or later may encounter a `build failed` error when installing Python through pyenv. 
+A patch was made available to remedy this:
+
+First, ensure that bzip2 and any other requirements for the Python build environment are met.
+See [pyenv wiki](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) for details:
+
+```
+brew install openssl readline sqlite3 xz zlib bzip2
+```
+
+Follow any additional steps that `brew` prompts for at the end of the installation. These  
+should include modifying path variables `LDFLAGS` and `CPPFLAGS`
+The commands from the `brew` output to modify the aforementioned path variables 
+can be placed in `~/.bash_profile` to make the change persistent
+
+Then install a patched Python 3.8.3 using `pyenv` by running:
+
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+Users of macOS 11 or later may encounter `pip` installation errors due to `pip` not being able to 
+locate the appropriate wheels.
+The information below will help remedy this:
```

Comment:
W

Reply:
@hannes-ucsc what did you mean when you commented `W` and `L` on some of the suggestions like this one and [here](https://github.com/DataBiosphere/azul/pull/3311#discussion_r705781746).

Reply:
See my pinned comment in `#team-boardwalk` on Slack.